tdrs-backend/tdpservice/users/api/login.py

```diff
@@ -291,6 +319,13 @@ def get_auth_options(self, access_token, sub):
         auth_options = {"login_gov_uuid": sub}
         return auth_options
 
+    def handle_email(self, user):
```

Comment:
I want to change the function name but am open to ideas as I don't feel this describes what we are accomplishing in a helpful way.

Reply:
verify_email? or verify_acf_email